test/cloudant.test.js

```diff
@@ -70,41 +70,6 @@ describe('cloudant connector', function() {
     });
   });
 
-  describe('updateAll and updateAttributes', function() {
```

Comment:
This test accomplishes the same suite of test in [`update.test.js`](https://github.com/strongloop/loopback-connector-cloudant/blob/master/test/update.test.js#L115)